opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/entrypoint/XS2aToFacadeMapper.java

```diff
@@ -0,0 +1,10 @@
+package de.adorsys.opba.protocol.xs2a.entrypoint;
+
+import de.adorsys.opba.protocol.api.dto.result.body.AccountListBody;
+import de.adorsys.xs2a.adapter.service.model.AccountListHolder;
+import org.mapstruct.Mapper;
+
+@Mapper(implementationPackage = "de.adorsys.opba.protocol.xs2a.entrypoint.generated")
```

Comment:
`de.adorsys.opba.protocol.xs2a.entrypoint.generated` should be a constant

Reply:
done